domain/build.gradle.kts

```
@@ -21,18 +22,20 @@ kotlin {
             }
         }
     }
-    ios {
-        binaries {
-            framework {
-                baseName = "domain"
-            }
-        }
+    ios()
+
+    cocoapods {
+        summary = "SongApp domian library."
+        homepage = "https://github.com/tooploox/KMMSongApp"
+        ios.deploymentTarget = "13.2"
```

14.2 is the newest I think
```suggestion
        ios.deploymentTarget = "14.2"
```